docs/description/description.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "patternId": "pylint",
+    "title": "Pylint tool",
+    "description": "Pylint is the most comprehensive static analysis tool for Python. It is extremely thorough and is the source of most messages that prospector outputs.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "pep8",
+    "title": "pep8 tool",
+    "description": "PEP 8 -- Style Guide for Python Code",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "pyflakes",
+    "title": "pyflakes tool",
+    "description": "A simple program which checks Python source files for errors.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "mccabe",
+    "title": "mccabe tool",
+    "description": "McCabe is a complexity checker for Python.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "dodgy",
+    "title": "dodgy tool",
+    "description": "Dodgy is a very basic toot to detect things such as accidental passwords or secret keys hard coded into files.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "pydocstyle",
+    "title": "pydocstyle tool",
+    "description": "pydocstyle is a static analysis tool for checking compliance with Python docstring conventions.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "pep257",
+    "title": "pep257 tool",
+    "description": "pep257 is a static analysis tool for checking compliance with Python PEP 257.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "frosted",
+    "title": "frosted tool",
+    "description": "Frosted is a fork of pyflakes that aims at more open contribution from the outside public, a smaller more maintainable code base, and a better Python checker for all.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "vulture",
+    "title": "vulture tool",
+    "description": "Vulture finds unused code in Python programs.",
+    "timeToFix": 5
+  },
+  {
+    "patternId": "pyroma",
+    "title": "pyroma tool",
+    "description": "",
```

Comment:
No description for this one? :( Maybe:

"pyroma gives a rating of how well a Python project complies with the best practices of the Python packaging ecosystem."